### PR TITLE
Site Redesign & README modification

### PR DESCRIPTION
**Pull request by colinrubbert**
Created a new branch for code review with all of the updated site redesigns.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-community.thefirehoseproject.com
```
Comment (kenmazaika) [inline CNAME:1]: I think we need this CNAME file to have the domain still work.

File: css/main.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+# Only the main Sass file needs front matter (the dashes are enough)
+---
+@charset "utf-8";
```
Comment (kenmazaika) [inline css/main.scss:4]: Does SCSS work in Jekyll without grunt?

**Comment by colinrubbert** [comment]
Right, I will copy that one over and re-push.

**Comment by kenmazaika** [comment]
Is http://colinrubbert.github.io/ still a place to see the latest changes, or to demo it should I pull it and run it locally?

**Comment by kenmazaika** [comment]
Made some comments, overall looks solid!

**Comment by colinrubbert** [comment]
The latest changes will be on this branch. I didn't change the grid system on my local copy or I can't remember if I did. I did make those grid changes on this branch though.

**Comment by kenmazaika** [comment]
For posterity sake, here's a screenshot of the post page.

screen shot 2015-08-14 at 12 16 52 pm

File: css/main.scss (hunk 2 above)
Comment (colinrubbert) [inline css/main.scss:4]: Yes it does. I was actually having a ton of issues with LESS and grunt in the old version, I had to manually go into the css file (which they recommend not to) and hard code the styling in there.

**Comment by kenmazaika** [comment]
Looks good to me.  Let me know if you want me to press the merge button, or have anything you want to add first. 

AFAIK this is perfect, but want to quickly check.

**Comment by colinrubbert** [comment]
You think we should trust github to merge or should I merge to master locally then push up? There's a ton of changes that happened. You've been using the built in merge pull requests much longer than I have, still a little hesitant.

**Comment by kenmazaika** [comment]
GitHub's merge button is reliable.  I'll press it! :)

**Comment by kenmazaika** [comment]
Took a moment to update on the site (I think stuff was cached), but it's live now.

Thanks for all the help, Colin! :)

**Comment by colinrubbert** [comment]
It took a bit on mine too but I'm super happy that it worked!

I'm happy to help out and make things easier for everyone, it was fun! It got slightly out of hand considering yesterday I was only going to spend 15 minutes updating some small nagging issues then turned into 8 hours of redoing the whole thing haha. 

I've got a list of things that can be done as student open source contributions. I will open issues for each of them and then we can tell the students if they want to help they can. I would be interested in your opinion on moving the bootstrap, font awesome, and jquery from cdn links to locally hosted.

**Comment by kenmazaika** [comment]
I'd probably keep them on the CDN.  Hosting it on the CDN is a bit faster, because people already have the file cached in their browser.

Having said that, some of the images are terribly un-optimized, so there's other more low hanging fruit.

**Comment by colinrubbert** [comment]
I think it is significantly easier to use the CDN stuff too. I agree the images are horrible optimized and that's one of the things on my list. I can shoot you an email or share a google doc of what I've come up with as issue submissions.

**Pull request merged**